WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -390,6 +355,7 @@ protected void onDestroy() {
 
         mDispatcher.unregister(this);
         super.onDestroy();
+        mBinding = null;
```

Comment:
Warn (⚠️): Just something I forgot to mention here, when using ViewBinding on the activity level, it is not necessary to nullify the binding during `onDestroy()`. This is only necessary on the fragment level, see docs, [activities](https://developer.android.com/topic/libraries/view-binding#activities) vs [fragments](https://developer.android.com/topic/libraries/view-binding#fragments):

```
Note: Fragments outlive their views. Make sure you clean up any references to the binding
class instance in the fragment's onDestroyView() method.
```

Reply:
Thanks for the above docs explanation, did the changes!

Reply:
Nice, thanks Neel! 👍